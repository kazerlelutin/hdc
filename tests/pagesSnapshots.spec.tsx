import { render, cleanup } from '@testing-library/react';
import Home from '../pages';
import MyApp from '../pages/_app';

describe('Pages snapshots', () => {

    afterEach(cleanup);

    it('snap <MyApp />', () => {
        const { asFragment } = render(<MyApp Component={Home} pageProps={{test:"test"}}/>);
        expect(asFragment()).toMatchSnapshot();
    });

    it('snap <Home />', () => {
        const { asFragment } = render(<Home/>);
        expect(asFragment()).toMatchSnapshot();
    });
});