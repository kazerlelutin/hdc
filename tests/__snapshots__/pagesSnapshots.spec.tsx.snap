// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages snapshots snap <Home /> 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="sand"
    >
      <div
        class="mountain one"
      />
      <div
        class="mountain two"
      />
      <div
        class="mountain three"
      />
      <div
        class="mountain four"
      />
      <div
        class="mountain five"
      />
      <div
        class="mountain six"
      />
      <div
        class="mountain seven"
      />
    </div>
    <div
      class="container"
    >
      <div
        class="rupteur"
        data-testid="rupteur"
        data-theme="light"
      >
        <div
          class="state"
        />
      </div>
    </div>
    <h1
      class="title"
    >
      Tu es en avance !
    </h1>
  </div>
</DocumentFragment>
`;

exports[`Pages snapshots snap <MyApp /> 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="sand"
    >
      <div
        class="mountain one"
      />
      <div
        class="mountain two"
      />
      <div
        class="mountain three"
      />
      <div
        class="mountain four"
      />
      <div
        class="mountain five"
      />
      <div
        class="mountain six"
      />
      <div
        class="mountain seven"
      />
    </div>
    <div
      class="container"
    >
      <div
        class="rupteur"
        data-testid="rupteur"
        data-theme="light"
      >
        <div
          class="state"
        />
      </div>
    </div>
    <h1
      class="title"
    >
      Tu es en avance !
    </h1>
  </div>
</DocumentFragment>
`;
